packages/postcss-syntax/src/parse.ts

```diff
@@ -39,7 +39,7 @@ export const parse = (css: string | { toString(): string }, opts?: ParserOptions
   Object.entries(cssEntries).forEach(([declarator, slots]) => {
     Object.entries(slots).forEach(([slot, rules]) => {
       cssRuleSlotNames.push(`${declarator} ${slot}`);
-      let cssRule = rules.join('');
+      let cssRule = rules.join('').replace(/\n/g, ' ');
```

Comment:
Can you pleause use [os.EOL](https://nodejs.org/api/os.html#os_os_eol) to handle windows/nix formats?